configs/approval.json

```diff
@@ -125,9 +175,33 @@
           "resourceDefinitions": [
             {
               "attributeFilter": {
-                "value": "*",
-                "key": "id",
-                "operation": "equal"
+                "key": "scope",
+                "operation": "equal",
+                "value": "user"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:actions:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "key": "scope",
+                "operation": "equal",
+                "value": "user"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:workflows:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "key": "scope",
+                "operation": "equal",
+                "value": "user"
```

Comment:
@hsong-rh Are the workflows created per user. This permission is saying that the logged in user would only be able to see the workflows he or she created. He/she would not be able to see all workflows in the tenant. If he/she is allowed to read all Workflows then the scope should be "admin"

Reply:
@mkanoor Approval service actually does not use `user scope`. The better way is only having permission here, no need to defined `resourceDef`, like we mentioned before.